src/library_path.js

```diff
@@ -66,8 +66,7 @@ addToLibrary({
     basename: (path) => {
       // EMSCRIPTEN return '/'' for '/', not an empty string
```

Comment:
Is there an extra quote here ?  Maybe remove it as part of this PR?

Reply:
Right, I'd removed it in https://github.com/emscripten-core/emscripten/pull/23136.